lib/oops_request_worker.rb

```diff
@@ -0,0 +1,7 @@
+class OopsRequestWorker < BaseWorker
+  sidekiq_options retry: 3, queue: 'default'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.